### ISSUE TITLE
error outputin huge file with buffer = 4096 )) just sintacs error

### ISSUE THREAD

**Issue by VasilijKolomiets** [spyder-ide/spyder#5810]
## Description

**What steps will reproduce the problem?**

1.  fw.write(out_text, buffer = 4096)    
2.  out_text was a huge string with '\n' inside
3.  Python 3.6   on windows

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "F:\Games\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 906, in <lambda>
    clear_variables))
  File "F:\Games\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 948, in execute_code
    sw.sig_prompt_ready.disconnect()
TypeError: disconnect() failed between 'sig_prompt_ready' and all its connections

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by CAM-Gerlach** [comment]
Presumably, it worked with a smaller file or a different buffer? That narrows down the problem to something specific to those properties, rather than a deeper issue.

In any case, unless this only fails using Spyder and not when using qtconsole, ipython, or python itself, I'm not sure how it would qualify as an issue with Spyder. In that case, you would want to report the bug at the lowest level it still occurs at, again assuming you've made sure it isn't something intentional or specific to your usage. Best of luck!

**Comment by ccordoba12** [comment]
This problem was already fixed and the fix will be available in our 3.2.5 version.

In the meantime, please avoid using dedicated consoles or install 3.2.3.
